cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -341,6 +341,7 @@ img_mgmt_upload_first_chunk(struct mgmt_ctxt *ctxt, const uint8_t *req_data,
 static int
 img_mgmt_upload(struct mgmt_ctxt *ctxt)
 {
+    struct mgmt_evt_op_cmd_status_arg arg;
```

Comment:
name to generic (as this is long routine): prefer `status_arg` or cmd_status_arg